swift/apple/Firezone/Application/FirezoneApp.swift

```diff
@@ -74,23 +74,6 @@ struct FirezoneApp: App {
     public var store: Store?
 
     func applicationDidFinishLaunching(_: Notification) {
-
```

Comment:
Since we don't need to migrate the firezone-id anymore per se, this can be removed.